Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -97,9 +99,32 @@ struct SettingsView: View {
                     }
                     
                     Button {
-                        isShowingAddVehicle.toggle()
+                        isShowingAddVehicle = true
                     } label: {
                         Text("Add Vehicle", comment: "Label to add a vehicle.")
+                    }.navigationDestination(isPresented: $isShowingAddVehicle) {
```

Comment:
This `.navigationDestination` should be on the `Form`, it shouldn't be on the `Button`
Put it before the `.alert`, so it's the first modifier on the `Form`